build.assets/Dockerfile

```
@@ -298,8 +298,10 @@ RUN go install github.com/google/addlicense@v1.0.0
 RUN go install github.com/daixiang0/gci@v0.9.1
 
 # Install golangci-lint.
-RUN curl -fsSL "https://raw.githubusercontent.com/golangci/golangci-lint/v1.52.2/install.sh" | \
-    sh -s -- -b "$(go env GOPATH)/bin"
+ARG GOLANGCI_VERSION # eg, "1.53.0"
+RUN TAG="v$GOLANGCI_VERSION" && \
+    curl -fsSL "https://raw.githubusercontent.com/golangci/golangci-lint/$TAG/install.sh" | \
+    sh -s -- -b $(go env GOPATH)/bin $TAG
```

We don't need to pipe in the version from the Makefile, it's only used here.

```suggestion
RUN TAG='v.53.0' && \
    curl -fsSL "https://raw.githubusercontent.com/golangci/golangci-lint/$TAG/install.sh" | \
    sh -s -- -b $(go env GOPATH)/bin "$TAG"
```